src/key.rs

```diff
@@ -212,9 +212,9 @@ impl SecretKey {
         SecretKey(sk)
     }
 
-    /// Serializes the secret key as byte value.
+    /// Gets the secret key as a byte value.
```

Comment:
"Returns" would sound more natural to me.

Reply:
Thanks, use as suggested.